Core/Object Arts/Dolphin/Base/Integer.cls

```diff
@@ -497,6 +497,23 @@ isPowerOfTwo
 
 	^self strictlyPositive and: [(self bitAnd: self - 1) = 0]!
 
+isPrime
```

Comment:
As a general point, when adding methods to base classes please follow the standard formatting for consistency. The easiest way to do this is to auto-format with Ctrl+W, or use Reformat/Accept (Ctrl+Shift+S) when saving. Unfortunately you may find this messes up your comment alignment.